packages/hdwallet-native/src/ethereum.ts

```diff
@@ -27,6 +28,15 @@ export function MixinNativeETHWalletInfo<TBase extends core.Constructor<core.HDW
       return true;
     }
 
+    async ethGetChainId(): Promise<number | null> {
```

Comment:
I am unsure about the necessity of doing this (enables the switch button on web, but has not real utility currently)